src/css/wikidot-structure.css

```diff
@@ -0,0 +1,3350 @@
+table.page-files tr[id*="file-row"] {
+	grid-template-columns: 1.5fr 1.5fr 1fr 2fr;
+	padding: 0.5em 0;
+}
+
+table.page-files tr[id*="file-options"] {
+	grid-template-columns: 0 0 0 1fr;
+	margin-top: 0;
+	padding: 0 0 0.5em;
+}
+
+table.page-files tr[id*="file-row"]:nth-of-type(odd),
+table.page-files tr[id*="file-row"]:nth-of-type(odd)+table.page-files tr[id*="file-options"],
+table.page-files tr[id*="file-row"]:nth-of-type(odd)+.highlight {
+	background-color: rgba(var(--swatch-tertiary-color), 0.2);
+}
+
+table.page-files tr[id*="file-options"]~tr[id*="file-row"]:nth-of-type(odd),
+table.page-files tr[id*="file-row"]:nth-of-type(even)+.highlight {
+	background-color: transparent;
+}
+
+table.page-files tr[id*="file-options"]~tr[id*="file-row"]:nth-of-type(even) {
+	background-color: rgba(var(--swatch-tertiary-color), 0.2);
+}
+
+table.page-files thead tr th,
+table.page-files tbody tr td {
+	display: flex;
+	position: relative;
+	align-items: center;
+	justify-content: center;
+}
+
+table.page-files th {
+	--wght: 600;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	width: 100%;
+	height: 100%;
+	background-color: rgb(var(--swatch-primary));
+	color: rgb(var(--swatch-text-secondary-color));
+	text-transform: uppercase;
+}
+
+table.page-files tr>td>a:any-link:not(.btn) {
+	--link-color: var(--swatch-text-general);
+	--visited-link-color: var(--swatch-text-general);
+	--wght: var(--ui-wght);
+	display: flex;
+	align-items: center;
+	padding: 0 0.5em;
+	transition:
+		color 160ms cubic-bezier(0.4, 0, 0.2, 1),
+		font-weight 160ms cubic-bezier(0.4, 0, 0.2, 1),
+		font-variation-settings 160ms cubic-bezier(0.4, 0, 0.2, 1),
+		box-shadow 200ms cubic-bezier(0.4, 0, 0.2, 1),
+		text-decoration 200ms cubic-bezier(0.4, 0, 0.2, 1);
+	background-color: transparent;
+	box-shadow: inset 0 0 0 0 rgb(var(--swatch-primary-darkest));
+	text-decoration: underline rgb(var(--swatch-primary-darkest)) 0.125em;
+}
+
+table.page-files tr>td>a:not(.btn):hover,
+table.page-files tr>td>a:not(.btn):active {
+	--wght: var(--ui-hvr-wght);
+	box-shadow: inset 0 -2em 0 0 rgb(var(--swatch-primary));
+	color: rgb(var(--swatch-text-secondary-color));
+	text-decoration: underline rgb(var(--swatch-primary)) 0;
+}
+
+table.page-files tr>td>a:not(.btn):focus-within {
+	--wght: var(--ui-hvr-wght);
+	box-shadow: inset 0 -2em 0 0 rgb(var(--swatch-primary));
+	color: rgb(var(--swatch-text-secondary-color));
+	text-decoration: underline rgb(var(--swatch-primary)) 0;
+}
+
+table.page-files tbody tr td:not(.page-rate-widget-box):not(#search-top-box-form)>a.btn {
+	margin: 0 0.5em;
+}
+
+#file-upload-form table,
+#file-upload-form table>*,
+#file-upload-form table>*>* {
+	display: contents;
+}
+
+#file-upload-form table.form {
+	display: grid;
+	grid-gap: 0.5em;
+	grid-template-columns: repeat(2, 1fr);
+}
+
+#file-upload-form table.form tr:nth-of-type(3) td {
+	grid-column: span 2;
+}
+
+/* Removing for now until autoprefixer preFIXES itself 

```

Comment:
You created an issue for this right? Can you link to the issue in the comment so people know what they're waiting on?

I would also briefly describe what the issue is.

Reply:
I did, yes! And can do.
I'm keeping the pun, though :p